tests/test_database.py

```diff
@@ -16,6 +16,13 @@ async def asyncSetUp(self) -> None:
         """Grab a JWT for all the tests to share."""
         if "REPLIT_DB_URL" in os.environ:
             self.db = AsyncDatabase(os.environ["REPLIT_DB_URL"])
+        elif "DB_RIDT" in os.environ:
```

Comment:
Is this var set exclusively in deployment?

Reply:
Just for testing, I found it to be the quickest way to switch between using JWT or Repl Identity for auth